patches/aspnetcore/0009-Support-global.json-on-arm64-as-well.patch

```diff
@@ -0,0 +1,33 @@
+From 84d274a8f3d416b0a5bd999e3d1c43ae1535e38f Mon Sep 17 00:00:00 2001
```

Comment:
This is https://github.com/aspnet/AspNetCore/pull/14790. **MERGED** into `master`.